app/src/page/channel/view.js

```diff
@@ -81,7 +84,7 @@ class ChannelPage extends React.PureComponent {
       <View style={channelPageStyle.container}>
         <PageHeader title={name} onBackPressed={() => navigateBack(navigation, drawerStack, popDrawerStack)} />
         {contentList}
-        {(totalPages > 1) &&
+        {(totalPages > 1) && this.state.showPageButtons &&
```

Comment:
Please add parens around the conditional code (and indent).
*Referring to the code below this line